packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -58,12 +59,26 @@ const DMT5TradeModal = observer(
                 platform: CFD_PLATFORMS.MT5,
                 is_eu: show_eu_related_content,
                 shortcode: getCompanyShortcode(),
+                product,
                 is_mt5_trade_modal: true,
             });
         const getAccountTitle = () => {
             if (show_eu_related_content) return 'CFDs';
             else if (mt5_trade_account.market_type === MARKET_TYPE.SYNTHETIC) return 'Standard';
-            else if (mt5_trade_account.market_type === MARKET_TYPE.ALL) return 'SwapFree';
+            else if (mt5_trade_account.market_type === MARKET_TYPE.ALL && product === PRODUCT.SWAPFREE)
```

Comment:
These titles, heading, icons seems to be always come in a set.. this modal is quite the horrible design

Reply:
We have initiative to re-write all of our components which have issues. These will be taken care of starting from the next sprint.

Reply:
Same for this